[2024-01-23T23:45:36.934+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tutorial_dag.load scheduled__2024-01-23T22:30:00+00:00 [queued]>
[2024-01-23T23:45:36.959+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tutorial_dag.load scheduled__2024-01-23T22:30:00+00:00 [queued]>
[2024-01-23T23:45:36.960+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-01-23T23:45:37.006+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load> on 2024-01-23 22:30:00+00:00
[2024-01-23T23:45:37.026+0000] {standard_task_runner.py:60} INFO - Started process 228 to run task
[2024-01-23T23:45:37.086+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tutorial_dag', 'load', 'scheduled__2024-01-23T22:30:00+00:00', '--job-id', '12', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/tutorial_dag.py', '--cfg-path', '/tmp/tmpb54u3j5l']
[2024-01-23T23:45:37.089+0000] {standard_task_runner.py:88} INFO - Job 12: Subtask load
[2024-01-23T23:45:37.382+0000] {task_command.py:423} INFO - Running <TaskInstance: tutorial_dag.load scheduled__2024-01-23T22:30:00+00:00 [running]> on host 4e1f8651ba76
[2024-01-23T23:45:37.612+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tutorial_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T22:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-23T22:30:00+00:00'
[2024-01-23T23:45:37.632+0000] {logging_mixin.py:188} INFO - {'total_order_value': 1236.7}
[2024-01-23T23:45:37.634+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T23:45:37.683+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=tutorial_dag, task_id=load, execution_date=20240123T223000, start_date=20240123T234536, end_date=20240123T234537
[2024-01-23T23:45:37.766+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T23:45:37.780+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
